openassessment/xblock/grade_mixin.py

```diff
@@ -374,7 +374,7 @@ def _get_assessment_part(title, feedback_title, part_criterion_name, assessment)
         if len(assessments) > 0:
             first_assessment = assessments[0]
             option = first_assessment['option']
-            if option and option.get('points'):
+            if option and option.get('points', None) != None:
```

Comment:
I think this fixes most of our problems with acceptance tests - `0 points` was being hidden because 0 is falsy.

<img width="305" alt="screen shot 2017-08-10 at 11 02 05 am" src="https://user-images.githubusercontent.com/7373924/29176995-82927120-7dbb-11e7-80e3-e88c56bc65e1.png">